convert_json_to_txt.py

```diff
@@ -0,0 +1,76 @@
+import pandas as pd
+import os
+import cv2
+import json
+
+# This script converts VIA (VGG image Annotator) json files to .txt files required for YOLOv3.
+
+def create_txt_files(via_json_file, images_folder, txt_dest_folder, attribute_name, labels):
+	"""
+	Input:
+	via_json_file = json files exported from VIA annotator
+	images_folder = path folder containing images
+	txt_dest_folder = destination folder for txt files
+	label_idx_dictionary = dictionary whose keys are label names
+						   in VIA annotator and values are the 
+						   corresponding integers to be use for 
+						   YOLOv3. For example: 
+						   {'label0': 0, 'label1':1, 'label3': 3}
+	attribute_name = attribute name assigned from VIA annotator
```

Comment:
It seems like I could use this script but maybe I need more context to understand what `attribute_name` is